triaging.rst

```diff
@@ -3,193 +3,247 @@
+
 Thus if an issue for a new feature is assigned for e.g., Python 3.7 but is not
 applied before Python 3.7.0 is released, this field should be updated to say
 Python 3.8 as the version and drop Python 3.7.
 
 Priority
 ''''''''
-How important is this issue?
-
-low
-    This is for low-impact bugs, or feature requests of little utility.
-normal
-    The default value for most issues, which deserve fixing but without
-    any urgency to do so.
-high
-    Make some effort to fix the issue before the next final release.
-critical
-    This issue should definitely be fixed before the next final release.
-deferred blocker
-    The issue will not hold up the next release, but will be promoted to a
-    release blocker for the following release, e.g., won't block the next
-    release of a1 but will block a2.
-release blocker
-    The issue must be fixed before *any* release is made, e.g., will block the
-    next release even if it is an alpha release.
+What is the severity and urgency?
+
++------------------+--------------------------------------------------------+
+| Priority         | Description                                            |
++==================+========================================================+
+| low              | This is for low-impact bugs.                           |
++------------------+--------------------------------------------------------+  
+| normal           | The default value for most issues filed.               |
++------------------+--------------------------------------------------------+
+| high             | Try to fix the issue before the next final release.    |
++------------------+--------------------------------------------------------+
+| critical         | Should definitely be fixed for next final release.     |
++------------------+--------------------------------------------------------+
+| deferred blocker | The issue will not hold up the next release, *n*. It   |
+|                  | will be promoted to a *release blocker* for the        |
+|                  | following release, *n+1*.                              |
++------------------+--------------------------------------------------------+
+| release blocker  | The issue **must** be fixed before *any* release is    |
+|                  | made, e.g., will block the next release even if it is  |
+|                  | an alpha release.                                      |
++------------------+--------------------------------------------------------+
 
 As a guideline, *critical* and above are usually reserved for crashes,
 serious regressions or breakage of very important APIs.  Whether a bug
-is a *release blocker* is a decision better left to the release manager so,
-in any doubt, add him or her to the *nosy list*.
+is a *release blocker* is decided by the release manager. Triagers may
+recommend this priority and should add the release manager to the 
+*nosy list*.
```

Comment:
Should we link to https://devguide.python.org/#status-of-python-branches here, and perhaps add the RM's tracker ID's to that table?

Reply:
Good suggestion